requirements.txt

```diff
@@ -7,20 +8,30 @@ dj-database-url==0.4.2
 Django==2.0.1
 django-debug-toolbar==1.9.1
 django-dotenv==1.4.2
+django-jet==1.0.7
 django-nose==1.4.5
 djangorestframework==3.7.7
 docopt==0.6.2
 flake8==3.5.0
 gunicorn==19.7.1
 idna==2.6
+isort==4.3.4
```

Comment:
Besides django-jet, I don't think we need these other python packages.

Reply:
on it.

Reply:
Done